### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by allanpersson**
Suggested change: URL pointing to recorder for Mariadb docs instead of returning to addon docs

### CODE REVIEW

File: mariadb/README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ In case you've found a bug, please [open an issue on our GitHub][issue].
 [hostname]: https://mariadb.com/kb/en/library/create-user/#host-name-component
 [grant]: https://mariadb.com/kb/en/library/grant
 [mariadb-hass]: https://www.home-assistant.io/addons/mariadb
```
Comment (frenck) [inline mariadb/README.md]: The `mariadb-hass` link reference is now unused, please remove it.

```suggestion
```

File: mariadb/README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ This should be the same database defined in `databases`.
 
 ## Home Assistant Configuration
 
-MariaDB will be used by the `recorder` and `history` components within Home Assistant. For more information about setting this up, see the [MariaDB][mariadb-hass] documentation for Home Assistant.
+MariaDB will be used by the `recorder` and `history` components within Home Assistant. For more information about setting this up, see the [MariaDB][mariadb-ha-recorder] documentation for Home Assistant.
```
Comment (frenck) [inline mariadb/README.md]: We should adjust the link title as well in this case:

```suggestion
MariaDB will be used by the `recorder` and `history` components within Home Assistant. For more information about setting this up, see the [recorder integration][mariadb-ha-recorder] documentation for Home Assistant.
```

**Comment by allanpersson** [comment]
Thanks @frenck i hope i dit it right accepting your changes :pray:

**Comment by frenck** [comment]
Looking good! Thanks, well spotted 👀

**Pull request merged**